### PR TITLE
[DOCS] Change index name in rollover ILM example

### PR DESCRIPTION
**Pull request by immon**
The warning section above the example tells that index name has to end with the digits but the example itself uses index name without digits which is confusing.

I understand that it meant to be a generic rollover example, but here we are in the context of ILM

### CODE REVIEW

**Review by jrodewig — APPROVED** [review]
LGTM but I'll wait for a review from a dev before merging.

Thanks @immon.

**Review by jakelandis — APPROVED** [review]
LGTM

**Pull request merged**
